dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/FATSuite.java

```diff
@@ -66,8 +66,6 @@
                 CheckpointWithSecurityManager.class,
                 SSLTest.class,
                 PasswordUtilsTest.class,
-                MPOpenTracingJaegerTraceTest.class,
```

Comment:
I assume you will also remove the test applications and test servers associated with these tests if they are no longer used by any of the remaining tests in this bucket?

Reply:
Yes, the MP test servers and applications have been removed from io.openliberty.checkpoint_fat